README.md

```diff
@@ -32,7 +32,12 @@ When the `avrogo` command generates Go datatypes from Avro schemas, it uses the
 - `["null", T]` encodes as `*T`
 - `[T, "null"]` encodes as `*T`
 - `[T₁, T₂, ...]` (a union) encodes as `interface{}` that should hold only the types for `T₁`, `T₂`, etc.
-- `{"type": "record", "name": "R", "fields": [....]}` encodes as a Go struct type named `R` with corresponding fields.
+- `{"type": "record", "name": "R", "fields": [....]}` encodes as a Go
+  struct type named `R` with corresponding fields.
+- `{"type": "long", "logicalType": "timestamp-micros"}` is represented
+  as `time.Time` type
+- `{"type": "string", "logicalType": "uuid"}` is represented as
+  [github.com/satori/go.uuid.UUID](https://pkg.go.dev/github.com/satori/go.uuid#UUID) type.
```

Comment:
I would advocate for using https://github.com/google/uuid

Reply:
I guess it might be reasonable to make that configurable, depending on what type is commonly in use in the code base being generated in.

Reply:
I agree that it could be configurable but it may be a second iteration of it :)